packages/design-picker/src/components/theme-card/style.scss

```diff
@@ -1,3 +1,6 @@
+@import "@automattic/typography/styles/variables";
+@import "client/assets/stylesheets/shared/mixins/_breakpoints";
```

Comment:
Hi @wojtekn! As this P2 p4TIVU-9lC-p2 mentions, we try to keep packages separated from Calypso client. @okmttdhr I'm not familiar to how Storybook works, but is it possible to import these files on Storybook's side?

Reply:
My bad, `_breakpoints.scss` was not needed in this file.